java/src/main/java/com/google/oak/verification/BUILD

```diff
@@ -29,6 +29,14 @@ java_binary(
     ],
 )
 
+java_binary(
+    name = "main_verifier",
```

Comment:
I think it could be called a `cli` or a `verifier_cli`

Alsom why do we need Java binaries if we just provide a verification library?

Reply:
Calling from Python is easiest via a command line. We had java_binary for log entry verification so it can be called from Python.